configure

```diff
@@ -2851,12 +2851,18 @@ fi
 
 
 #!/bin/sh
-if test -d ./src/boost.tar.gz;
```

Comment:
IMO there was a bug here. The old code was checking for a folder/directory with the name of the **file**.

Reply:
I agree.